dev/TOC.md

```diff
@@ -289,6 +289,7 @@
       - [常见错误修复](/dev/reference/tidb-binlog/troubleshoot/error-handling.md)
     - [FAQ](/dev/reference/tidb-binlog/faq.md)
   + 周边工具
+    - [工具使用指南](/dev/reference/tools/use-guide.md)
```

Comment:
The file name is a little bit weird to me. Suggest using `user-guide.md`. But let's decide based on #2244.

Reply:
Okay. I'll open a PR to change it. Later, you can make a comment in #2244 to suggest changing the file name.

Reply:
@yikeke Thanks. Just hold on. Maybe we can resolve it in #2244.

Reply:
Sure.

Reply:
@yikeke FYI. I updated the file name in https://github.com/pingcap/docs-cn/pull/2244/commits/42431caef6538a3f7a9b5d55de22a0b20f3f4517 of #2244.

Reply:
Okay. @toutdesuite will align this PR. Please remember to change the file name.